Fixed probability in Meland mirror

HomesGH opened the pull request:
The option to set fixed probability in Meland mirror was added

SteffenSeckler commented on src/plugins/Mirror.cpp:
```diff
@@ -154,17 +154,26 @@ void Mirror::readXML(XMLfileUnits& xmlconfig)
 	{
 		_melandParams.use_probability_factor = true;
-		_melandParams.velo_target = 0.4;
+		_melandParams.velo_target = 4.4;
```
did you intend to change this value?

HomesGH replied:
Actually the best way would be to remove this line (and the above). The right value is assigned to the variable afterwards and if not the program is aborted anyway. See lines 160 - 166.

SteffenSeckler commented:
once this pull request is ready to merge, please request (or re-request) a review on the top right.

HomesGH commented:
I changed the variable assignment mentioned above plus added/changed the following things: 

Major:
  - Added ramping method to the mirror, which can be used to set the reflection ratio dependent on the current simulation step
 Minor:
  - Data type changed from auto to "uint32_t" to prevent a warning (line 31/32)
  - Correction of a typo

The review is re-requested

SteffenSeckler commented on src/plugins/Mirror.cpp:
```diff
@@ -153,18 +154,58 @@ void Mirror::readXML(XMLfileUnits& xmlconfig)
 	if(MT_MELAND_2004 == _type)
 	{
-		_melandParams.use_probability_factor = true;
-		_melandParams.velo_target = 0.4;
```
don't delete this default value!
currently, if meland/velo_target is not specified in the xml, this value is undefined and can be anything.
It would be good practice to initialize the _melandParams with some meaningful values inside of the constructor, or using inline variable initialization (inside of the header file, by adding the default value in curly brackets like this):
```c++
	struct MelandParams {
		bool use_probability_factor {false};
		double velo_target {0.4};
		float fixed_probability_factor {-1};
	} _melandParams;
```

HomesGH replied:
Ok. I added the default values during initializiation in the header-file as suggested.
But if meland/velo_target is not specified in the xml, wouldn't the whole program be aborted?

HomesGH replied:
As you mentioned, leaving the variables uninitialized seems like its not good practice, even though they are overwritten afterwards. Thanks for the hint!

SteffenSeckler replied:
true, it would be aborted, but it's better anyways, especially in case this plugin is used in unit-tests, etc.

SteffenSeckler commented:
will merge, once jenkins completes :)
